src/Pages/Ques/Ques.jsx

```diff
@@ -39,40 +47,82 @@ export function Ques() {
 
   function answerHandler(e) {
     setTimeout(() => {
-      setSelectedAnswer("");
-      setIndex((prevIndex) => (prevIndex < 2 ? prevIndex + 1 : navigate("/")));
+      setIndex((prevIndex) => {
+        if (prevIndex < 2) {
+          return prevIndex + 1;
+        } else {
+          navigate("/Score");
+          return prevIndex+1
+        }
+      });
     }, 1000);
```

Comment:
You can also take thig function outside of the component. in seprate folder may be.